src/modules/task/dto/get-task.dto.ts

```diff
@@ -0,0 +1,31 @@
+import { Type } from 'class-transformer';
+import { IsNumber, IsOptional } from 'class-validator';
+
+export enum TaskSortByEnum {
+  id = 'id',
+  title = 'title',
+  description = 'description',
+}
+
+export enum TaskSortDirection {
+  asc = 'asc',
+  desc = 'desc',
+}
+
+export class PaginationDto {
+  @IsNumber()
+  @Type(() => Number)
+  @IsOptional()
+  limit: number;
+
+  @IsNumber()
+  @Type(() => Number)
+  @IsOptional()
+  offset: number;
+
+  @IsOptional()
+  sortBy: TaskSortByEnum;
+
+  @IsOptional()
+  sortDirection: TaskSortDirection;
```

Comment:
Не хватает палидаций, сейчас это поле вообще никак не валидируется и в sortBy/sortDirection можно отправить что угодно, любое число или вообще любую строку
```suggestion
  @IsOptional()
  @IsEnum(TaskSortByEnum)
  sortBy: TaskSortByEnum;

  @IsOptional()
  @IsEnum(TaskSortDirection)
  sortDirection: TaskSortDirection;
```